prog/vnet/subnet_nexus.rb

```diff
@@ -38,13 +38,90 @@ def wait
       hop :refresh_mesh
     end
 
+    when_refresh_keys_set? do
+      ps.update(state: "refreshing_keys")
```

Comment:
Looking at this more, not so much this line, but it really seems like `ps` ought to be a `subject_is`, right? I missed this in 1ad6f54a62dd288dda4c8924a3e51246873096bb

Reply:
fixed it here https://github.com/ubicloud/ubicloud/pull/448